package.json

```diff
@@ -157,7 +157,7 @@
     "pm2-runtime": "./bin/pm2-runtime"
   },
   "dependencies": {
-    "async": "1.5",
+    "async": "^2.4.1",
```

Comment:
From the test log it fail inside the `/home/travis/build/Unitech/pm2/test/fixtures/inside/inner_restart.sh`  file on `restarted status should be one` test but i'll re-run in case its a false positive.